lib/internal/Magento/Framework/View/Layout/CacheKeyInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\View\Layout;
+
+/**
+ * Interface CacheKeyInterface
+ */
+interface CacheKeyInterface
```

Comment:
Please consider more specific naming.

Reply:
Ok, I'll try to find a more specific name.

Reply:
Before refactoring, I'd like to ask, LayoutCacheKeyInterface sounds like an appropiate name?